### PR TITLE
issue/1963bis

### PR DESCRIPTION
**Pull request by Frankaus**

### CODE REVIEW

**Comment by pimato** [comment]
@Frankaus lets put the Omen icon + Name vertical centered, currently:
Bildschirmfoto 2021-06-10 um 16 39 51

**Comment by Mi-Lan** [comment]
@Frankaus that margin geronimo mentioned is some margin put on with this media query just remove it and should be gtg 
Screenshot 2021-06-11 at 14 54 56

File: app/src/components/common/layout/header/index.tsx [hunk 1]
```diff
@@ -17,20 +17,26 @@ import { IconAdd, IconClose, IconOmen } from '../../icons'
...
-  }
+  border-bottom: 1px solid ${props => props.theme.buttonPrimaryLine.borderColorDisabled};
+
+  // As per the new Design specification this is now obsolete, 
+  // shall we keep it or leave it with the same metrics as above? 
+  // (same for HeaderInner below) 
+  // @media (min-width: ${props => props.theme.themeBreakPoints.md}) {
+  //   height: 100px;
+  //   align-items: center;
+  // }
```
Comment (Mi-Lan) [inline app/src/components/common/layout/header/index.tsx:39]: yeah I think its fine deleting it

File: app/src/components/common/layout/header/index.tsx [hunk 2]
```diff
@@ -41,13 +47,13 @@ export const HeaderInner = styled.div`
...
+  padding: 24px 0px;
   position: relative;
   width: ${props => props.theme.themeBreakPoints.xxl};
 
-  @media (min-width: ${props => props.theme.themeBreakPoints.md}) {
-    padding: 0 ${props => props.theme.paddings.mainPadding};
-  }
+  // @media (min-width: ${props => props.theme.themeBreakPoints.md}) {
+  //   padding: 30px 10px;
+  // }
```
Comment (Mi-Lan) [inline app/src/components/common/layout/header/index.tsx:56]: this as well...well probably do refactor of those in next release so we should keep it clean!

File: app/src/components/common/layout/header/index.tsx (hunk 1 above)
Comment (Frankaus) [reply app/src/components/common/layout/header/index.tsx:39]: cool I did

File: app/src/components/common/layout/header/index.tsx (hunk 2 above)
Comment (Frankaus) [reply app/src/components/common/layout/header/index.tsx:56]: yes sure I did too

**Review by pimato — APPROVED** [review]

**Review by Mi-Lan — APPROVED** [review]
Good job!

**Pull request merged**